includes/theme-navigation.php

```diff
@@ -59,18 +59,21 @@ public function theme_menus() {
 	 */
 	public function abbridged_menu_classes( $classes, $item, $args ) {
 		$post = get_post();
-		$current_or_parent_page = array_intersect( array( 'current-menu-item', 'current_page_parent' ), $classes );
-		$current_page_parent = ( $item->object_id == $post->post_parent );
-		$current_page_not_in_menu = ! in_array( 'current_page_parent', $classes, true );
+		$current_or_parent_menu_item = array_intersect( array( 'current-menu-item', 'current_page_parent' ), $classes );
+		$parent_of_page_not_in_menu = ( $item->object_id == $post->post_parent ) && ! in_array( 'current-page-parent', $classes, true );
```

Comment:
This one is intentional :) - see https://github.com/washingtonstateuniversity/WSUWP-spine-parent-theme/pull/325/commits/3a6d37e86bf3a9de4f722eafeff7983d6be61f7d